packages/actor-query-operation-orderby-sparqlee/lib/ActorQueryOperationOrderBySparqlee.ts

```diff
@@ -53,6 +53,7 @@ export class ActorQueryOperationOrderBySparqlee extends ActorQueryOperationTyped
         try {
           const result = await evaluator.evaluate(bindings);
           transformedStream._push({ bindings, result });
+          
```

Comment:
A good practise before doing PRs, is to check your diffs. Changes like these should not be needed, and just add unneeded noise to the PR.